oci_go_image/BUILD.bazel

```diff
@@ -47,8 +43,8 @@ platform_transition_filegroup(
     }),
 )
 
-# $ bazel build app:tarball
-# $ docker load --input $(bazel cquery --output=files app:tarball)
+# $ bazel build :tarball
```

Comment:
Updated these so that they would actually run. There is no `app` folder in this context.